liota/dccs/iotcc.py

```diff
@@ -72,17 +81,23 @@ def __init__(self, con):
         elif not self.comms.identity.password:
             log.error("Password not found")
             raise ValueError("Password not found")
-        thread = threading.Thread(target=self.comms.receive)
-        thread.daemon = True
+        recv_thread = threading.Thread(target=self.comms.receive)
+        recv_thread.daemon = True
         # This thread will continuously run in background to receive response or actions from DCC
-        thread.start()
+        recv_thread.start()
         # Wait for Subscription to be complete and then proceed to publish message
         time.sleep(0.5)
         self._iotcc_json = self._create_iotcc_json()
         self._iotcc_json_load_retry = int(read_liota_config('IOTCC_PATH', 'iotcc_load_retry'))
         self.enable_reboot_getprop = read_liota_config('IOTCC_PATH', 'enable_reboot_getprop')
         self.counter = 0
         self.recv_msg_queue = self.comms.userdata
+        self.req_ops_lock = Lock()
```

Comment:
Ideally, we should change all these variables to be internal only to this class (_req_ops_lock instead of req_ops_lock), but it will require changes to existing code as well. Hence, can be delayed to refactoring changes. Currently, if you want, can change the newly introduced one's to use _.

Reply:
Fixed. 

For others param will handle later.